src/main/java/org/openstreetmap/atlas/generator/tools/spark/SparkJob.java

```diff
@@ -53,7 +53,7 @@
             StringConverter.IDENTITY, Optionality.OPTIONAL);
     public static final Switch<String> OUTPUT = new Switch<>("output",
             "Output path of the Spark Job", StringConverter.IDENTITY, Optionality.REQUIRED);
-    public static final Switch<String> MASTER = new Switch<>("master", "The spark master URL",
+    public static final Switch<String> CLUSTER = new Switch<>("cluster", "The spark cluster URL",
```

Comment:
I think we should keep both, and have logic below to find which one was populated... That way downstream scripts do not fail right away?

Reply:
hmm.... I don't think that's a great idea, because there's no way of marking a parameter deprecated. so from a user standpoint, if they're not reading our code, it'll seem just as sudden when we finally remove the parameter

Reply:
I guess we could update the description string, but it seems pretty unlikely any downstream scripts are going to read that

Reply:
Ok, we can leave this as is then!